### DISCUSSION TITLE
Skip Feature

### DISCUSSION THREAD

**Discussion by otterboros** [nathanhoad/godot_dialogue_manager#302] [Q&A]
Hello! I'm getting a ton of use out of this dialogue manager for my new Godot 4 game. Thanks so much for making it!

I've been attempting to add a skip feature, similar to that seen in Renpy, where pressing CTRL advances each line rapidly and automatically advances to subsequent lines. This is mostly for internal use, so I can advance through story beats rapidly while debugging, but I do want the end product to be a feature in the playable build.

I'm wondering if anyone else has attempted it and, if so, how they went about implementing it?

**Comment by nathanhoad** [comment]
You could implement that in your balloon as each new line is read in from the Dialogue Manager - it could check for a toggle (or Ctrl being pressed) and then just skip to the next line.
